### ISSUE TITLE
Format of dataset

### ISSUE THREAD

**Issue by zeng-lingyun** [HobbitLong/CMC#57]
Hi,  Could you provide the format of the training set and validation set ?

**Comment by HobbitLong** [comment]
It's the standard ImageNet folders. If you are not familiar, I suggest you to follow PyTorch practice.

**Comment by zeng-lingyun** [comment]
Thanks. I have got the answer. :)
